.github/workflows/reports-scheduler-test-and-build-workflow.yml

```diff
@@ -47,5 +47,85 @@ jobs:
       - name: Upload Artifacts
         uses: actions/upload-artifact@v1
         with:
-          name: reports-scheduler
+          name: reports-scheduler-linux
+          path: reports-scheduler-builds
+
+  windows-build:
+    strategy:
+      matrix:
+        java:
+          - 11
+          - 17
+    runs-on: windows-latest
+
+    steps:
+      - name: Set up JDK ${{ matrix.java }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+
+      # reports-scheduler
+      - name: Checkout Reports Scheduler
+        uses: actions/checkout@v2
+
+      - name: RunBackwards Compatibility Tests
+        run: |
+          cd reports-scheduler
+          echo "Running backwards compatibility tests ..."
+          ./gradlew.bat bwcTestSuite
```

Comment:
Getting the following error, is this available? : 
`Could not HEAD 'https://artifacts.opensearch.org/releases/core/opensearch/1.1.0/opensearch-min-1.1.0-windows-x64.zip'. Received status code 403 from server: Forbidden`

Reply:
Same thing is happening for windows and macos

Reply:
see https://github.com/opensearch-project/observability/pull/1108#discussion_r993856324